### PR TITLE
Rbac/split backups from cluster

### PR DESCRIPTION
**Pull request by tsmith023**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [x] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: openapi-specs/schema.json [hunk 1]
```diff
@@ -27,6 +27,22 @@
       "type": "object",
       "description": "permissions attached to a role.",
       "properties": {
+        "backup": {
+          "type": "object",
+          "description": "resources applicable for backup actions",
+          "properties": {
+            "backend": {
```
Comment (moogacs) [inline openapi-specs/schema.json:34]: **suggestion (if-minor):** i think woe could do enum here for allowed backends , wdyt ?

Comment (tsmith023) [reply openapi-specs/schema.json:34]: Hmm, `backend` isn't an enum in other areas of `schema.json` so perhaps it's better to maintain consistency here. WDYT?

File: test/acceptance/authz/rbac_viewer_test.go [hunk 2]
```diff
@@ -43,9 +43,9 @@ func TestViewerEndpoints(t *testing.T) {
 		{endpoint: "authz/users/id/roles", methods: []string{"GET"}, success: []bool{true}, arrayReq: false},
 		{endpoint: "authz/users/id/assign", methods: []string{"POST"}, success: []bool{false}, arrayReq: false},
 		{endpoint: "authz/users/id/revoke", methods: []string{"POST"}, success: []bool{false}, arrayReq: false},
-		{endpoint: "backups/backend", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
-		{endpoint: "backups/backend/id", methods: []string{"GET", "DELETE"}, success: []bool{true, false}, arrayReq: false},
-		{endpoint: "backups/backend/id/restore", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
+		// {endpoint: "backups/backend", methods: []string{"GET", "POST"}, success: []bool{true, false}, arrayReq: false},
```
Comment (moogacs) [inline test/acceptance/authz/rbac_viewer_test.go:46]: **question (if-minor):** is this meant to be commented out ?

Comment (tsmith023) [reply test/acceptance/authz/rbac_viewer_test.go:46]: No! Thanks for the catch 😁

**Review by moogacs — APPROVED** [review]

**Pull request merged**